### ISSUE TITLE
Reload Filter on Rezone

### ISSUE THREAD

**Issue by ChrisK15** [ChaosRecipeEnhancer/ChaosRecipeEnhancer#640]
**Is your feature request related to a problem? Please describe.**
Not an issue.

**Describe the solution you'd like**
Just like the auto-fetch, integrate auto loot filter reload on rezone

**Describe alternatives you've considered**
Ive been using the hotkey, but it would be a big QOL to get this automated on rezone

**Additional context**
None

**Comment by HiMarioLopez** [comment]
Yeah this one is tough - I feel comfortable with the Auto-Fetch or Rezone for API calls since they're technically out-of-game but and don't leave any grey area for TOS violation

This is a bit trickier to navigate with reload filter on rezone since it's multiple in-game operations automated (e.g. BOTTING). Can't say I would ever want to include this to avoid risking anyone getting banned due to ToS violations

See here for GGG's rules on this: 

https://www.pathofexile.com/developer/docs#policy

> Macros must be invoked manually by the user (automated invocations such as but not limited to: timers, reacting to file changes, or from reading the screen are not allowed).
